.github/workflows/devcontainer_validate.yaml

```diff
@@ -0,0 +1,23 @@
+name: Devcontainer (Validate)
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize]
+    paths:
+      - ".github/workflows/devcontainer_validate.yaml"
+      - ".devcontainer/features/**"
+      - ".devcontainer/templates/**"
```

Comment:
Branch main?

Reply:
this is the `code review` action for devcontainers, it should run for the push on every branch